db/migrate/20220809191656_add_ssl_ca_to_http_proxy.rb

```diff
@@ -1,4 +1,4 @@
-class AddSslCaToHttpProxy < ActiveRecord::Migration[6.1]
+class AddSslCAToHttpProxy < ActiveRecord::Migration[6.1]
```

Comment:
```suggestion
class AddSSLCAToHTTPProxy < ActiveRecord::Migration[6.1]
```

It looks weird, but it would probably be correct.

Reply:
```
NameError: uninitialized constant AddSslCAToHttpProxy
Did you mean?  AddSSLCAToHTTPProxy
```

Reply:
So we're missing some more acronyms?

Reply:
I suppose.